code/game/objects/effects/lobby_art.dm

```diff
@@ -0,0 +1,46 @@
+GLOBAL_LIST_EMPTY(lobby_art_files)
+
+/obj/effect/lobby_art
+	name = "Lobby"
+	icon = 'icons/lobby art/tgmc_flag.dmi'	//Default icon
+	layer = FLY_LAYER
+	pixel_x = -64
+
+/obj/effect/lobby_art/New(loc, previous_art_icon)
+	..()
+	pick_art(previous_art_icon)
+
+/obj/effect/lobby_art/Initialize(mapload)
+	. = ..()
+	if(!mapload)	//Only fade in art that is loaded after the first
+		fade_in()
+	return INITIALIZE_HINT_LATELOAD
+
+/obj/effect/lobby_art/LateInitialize()
+	. = ..()
+	if(length(GLOB.lobby_art_files) > 1)	//Don't bother changing the art if there's only one
+		addtimer(CALLBACK(src, PROC_REF(transition_art)), CONFIG_GET(number/lobby_art_duration) SECONDS)
+
+///Create a new lobby art object and delete this one
+/obj/effect/lobby_art/proc/transition_art()
+	new /obj/effect/lobby_art(loc, icon_state)
+	layer -= 0.1	//I could use initial() but I don't see an instance to where anything should be changing the layer of it (minus this)
+	QDEL_IN(src, CONFIG_GET(number/lobby_art_fade_duration) SECONDS)
```

Comment:
Im denying the fade, we already have animated lobby UIs that swap and I dislike how it looks, people should be able to look at one

Reply:
That is quite literally just your opinion. I think it looks great. The animated bit is also silly, they still play out completely within 30 seconds. I could also just isolate the animated ones so the art doesn't transition if an animated one is selected roundstart if it really bugs you.

TM it and if the majority of people hate it then I can just close it. 🤷

Reply:
Who died and made you the aesthetics police???